iceberg/iceberg-handler/src/test/java/queries/negative/create_iceberg_table_failure.q

```diff
@@ -0,0 +1,2 @@
+set hive.vectorized.execution.enabled=true;
```

Comment:
Can we add here other table creation failure scenarios too? One that comes to mind now is where you specify a PARTITIONED BY clause but also specify the json spec in the table properties. Another one could be related to type mismatches, i.e. using an unsupported Hive type, like interval in the columns

Reply:
I suppose this was not meant to be comprehensive but just to set up the iceberg qtest infra? In that case, we can tackle these in upcoming PRs that beef up the test cases.

Reply:
Yes, this PR provides the skeleton to run additional q tests from upcoming PRs.